packages/driver/src/cy/commands/querying.coffee

```diff
@@ -380,6 +380,223 @@ module.exports = (Commands, Cypress, cy, state, config) ->
         ## always restore contains in case
         ## we used a regexp!
         restoreContains()
+  
+    shadowGet: (subject, selector, options = {}) ->
```

Comment:
not sure if this should work more like `get`, but I didn't understand how to leverage `subject`.